core/src/loader.rs

```diff
@@ -2307,11 +2342,11 @@ impl<'gc> Loader<'gc> {
 
                 match dialog_result {
                     Ok(dialog_result) => {
-                        use crate::avm1::globals::as_broadcaster;
+                        if let FileDialogResult::Selection(dialog_result) = dialog_result {
```

Comment:
I think `dialog_result` here is no longer a great fit, it should be more similar to `file_selection_group` (but possibly shorter).